specs/Checkbox.md

```diff
@@ -173,7 +147,24 @@ Removing the following two props because the ARIA spec dictates role='checkbox'
 
 Props being removed:
 
-ariaPoisitionInSet and ariaSetSize - when writing parent component, user should set these on the checkbox.
+ariaPoisitionInSet and ariaSetSize - when writing parent component, user should set these on the checkbox. ARIA spec dictates role='checkbox' doesn't need aria-posinset and aria-setsize. These are only valid for role='option' which is only in the case the checkbox is a part of a listbox, which is not something we need to account for in the base component API. If the user does need to provide these two props, slotProps could be used to apply additional props to any slot. 
+
+Note: rtl, styles, and theme come from compose or the ThemeProvider. And name has been added to support checkbox in form scenarios.
+
+### Work items and tracking progress on conversion from the current Fluent (Stardust) checkbox to the new Fluent checkbox we're building with this spec: 
+- [ ] Remove `animation` prop.
+- [ ] Remove `accessibility` prop.
+- [ ] Add `indeterminate` and `defaultIndeterminate` props to support the third state.
+- [ ] Redesign `keytipProps` from what's currently in Fabric and add the new implementation.
+- [ ] Decide if, in keeping label, it will be of type `ShorthandValue<TextProps>`and if other type, implement that.
+- [ ] Decide if, in keeping icon, it will be of type `ShorthandValue<IconProps>` and if other type, implement that.
+- [ ] Change `onChange` type to native `onChange?: (ev?: React.FormEvent<HTMLElement | HTMLInputElement>, checked?: boolean) => void;` from current `ComponentEventHandler`.
+- [ ] Decide if `onClick` is too similar to `onChange` and if so, remove it. If not, change its type to native instead of the current `ComponentEventHandler`.
+- [ ] Remove `toggle` - it will be a separate variant component of Checkbox.
+- [ ] Remove `variables` - theming will be handled differently in the new Fluent.
+- [ ] Remove `design` prop.
+- [ ] Decide on the TBD props `as`, `styles`, and `theme` which will apply to other components across the new Fluent library.
+- [ ] Keep `labelPosition`, `checked`, `defaultChecked`, `className`, and `disabled` props unchanged.
```

Comment:
The docs don't really mention that unrecognized native "div" (?) props should be mixed into root. I believe we expected that to continue to work.

Reply:
Do you mean the spec & not the docs? It's mentioned like this under the screenreader accessibility section but I'll add it here too: 

> should mix in native props expected for the element type defined in `as`.